webpack.config.js

```diff
@@ -35,7 +37,7 @@ const vendorModules = Object.keys(packageJSON.dependencies)
 
 const plugins = [
   new webpack.DefinePlugin({
-    'process.env.NODE_ENV': `"${process.env.NODE_ENV || 'dev'}"`,
```

Comment:
Also, this was a bad default. We do use "development" as the value and not "dev"